internal/pkg/vaultworker/vault-worker.go

```diff
@@ -137,7 +136,7 @@ func vaultInit(config *tomlConfig, httpClient *http.Client, debug bool) (sCode i
 	}
 
 	// Save the JSON structure to a file system JSON file
-	err = ioutil.WriteFile(config.SecretService.TokenFolderPath+"/"+config.SecretService.VaultInitParm, initRequestResponseBody, 0600)
+	err = ioutil.WriteFile(config.SecretService.TokenFolderPath+"/"+config.SecretService.VaultInitParm, initRequestResponseBody, 0644)
```

Comment:
why `0644` and not `0600` like it was before? isn't this the root token and we want it to only be root readable while it exists?

Reply:
This was due to the original plan for edgex-mongo container. Will change it back to 0600.

Reply:
thank you for changing it back